polkadot/xcm/src/v5/traits.rs

```diff
@@ -305,11 +305,11 @@ pub trait ExecuteXcm<Call> {
 	) -> Outcome {
 		let pre = match Self::prepare(message) {
 			Ok(x) => x,
-			Err(_) => return Outcome::Error { error: Error::WeightNotComputable },
+			Err(_) => return Outcome::Error { error: Error::WeightNotComputable, index: 0 },
```

Comment:
need to fix the `index` here, it should say which instruction was not weighable

Reply:
Would need the index in the source so I can bubble it up. Asked about it in https://github.com/paritytech/polkadot-sdk/pull/8535

Reply:
I don’t understand. Can you not change to get the index?